HousingRepairsSchedulingApi/UseCases/BookAppointmentUseCase.cs

```diff
@@ -15,15 +16,18 @@ public BookAppointmentUseCase(IAppointmentsGateway appointmentsGateway)
         }
 
         public Task<string> Execute(string bookingReference, string sorCode, string locationId,
-            DateTime startDateTime, DateTime endDateTime)
+            DateTime startDateTime, DateTime endDateTime, string orderComments)
         {
             Guard.Against.NullOrWhiteSpace(bookingReference, nameof(bookingReference));
             Guard.Against.NullOrWhiteSpace(sorCode, nameof(sorCode));
             Guard.Against.NullOrWhiteSpace(locationId, nameof(locationId));
+            Guard.Against.NullOrWhiteSpace(orderComments, nameof(orderComments));
             Guard.Against.OutOfRange(endDateTime, nameof(endDateTime), startDateTime, DateTime.MaxValue);
 
+            Helper.VerifyLengthIsUnder255Characters(orderComments);
```

Comment:
`GuardClauses` could be used here:
```suggestion
            Guard.Against.OutOfRange(orderComments.Length, nameof(orderComments), 0, 255);
```

Using this would reduce the need for the helper class and reduce our code base.